### PR TITLE
feat(QTREES-358): Add rainfall data to list in treeview

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds rain data to the tree view.

### CODE REVIEW

File: locales/de/common.json [hunk 1]
```diff
@@ -63,9 +64,54 @@
     "copiedFeedback": "In Zwischenablage kopiert!"
   },
   "treeView": {
+    "mapAriaLabel": "Kartenansicht",
+    "soilViz": {
+      "depthAriaLabel": "Tiefe: {{depth}} cm, Wasserversorgung: {{waterSupply}}",
```
Comment (dnsos) [inline locales/de/common.json:69]: Great, thanks for extracting all those texts!

File: pages/trees/[id].tsx [hunk 2]
```diff
@@ -82,54 +84,75 @@ const InfoList: FC<{
...
+}> = ({ treeData, nowcastData, nowcastIsLoading, nowcastError, rainData }) => {
+  const { t } = useTranslation('common')
   const averageStatusId =
     nowcastData &&
     !nowcastIsLoading &&
     !nowcastError &&
     nowcastData[3].value &&
     mapSuctionTensionToStatus(nowcastData[3].value)?.id
 
+  const totalRain = rainData?.reduce(
```
Comment (dnsos) [inline pages/trees/[id].tsx:99]: _If_ we do the calculation in the frontend, I would suggest moving this to the `useRainData` hook and returning a `sum` from there. What do you think?

File: src/lib/requests/getRainData.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { getBaseUrl } from '@lib/utils/urlUtil'
+
+type RawRainDataType = {
+  weekday: string
```
Comment (dnsos) [inline src/lib/requests/getRainData.ts:4]: I know this is still in discussion, but we should remember to update this once the PR for rain API route is finalized.

File: src/lib/requests/getRainData.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+): Promise<RainDataType[] | undefined> => {
+  if (!treeId) return
+
+  const REQUEST_URL = `${getBaseUrl()}/api/trees/rainfall`
+
+  const REQUEST_PARAMS = new URLSearchParams({
+    [TREE_ID_COLUMN_NAME]: `${treeId}`,
+  })
+
+  const response = await fetch(`${REQUEST_URL}?${REQUEST_PARAMS.toString()}`)
```
Comment (dnsos) [inline src/lib/requests/getRainData.ts:31]: Ahhh, adding the `.toString()` is the trick to make TypeScript happy. I had been wondering that in the nowcast and forecast requests.

**Review by dnsos — COMMENTED** [review]
Great work, very clean!

There are probably some changes to do once #51 is merged, but other than that this looks good 👍

File: pages/trees/[id].tsx (hunk 2 above)
Comment (vogelino) [reply pages/trees/[id].tsx]: We now do the calculation on the API side so this isn't necessary anymore.

**Pull request merged**